app/styles/main.scss

```diff
@@ -1119,3 +1094,56 @@ input[type=radio]:checked + label:before, input[type=checkbox]:checked + label:b
     background-color: #ccc;
 }
 
+.direct-descendants-margin {
+  :first-child {
+    margin-left: 0;
+  }
+
+  :last-child {
+    margin-right: 0;
+  }
+
+  & > * {
+    margin: 0 2px;
+  }
+}
+.flex-column {
+  display: flex;
+  flex-direction: column;
+}
+.flex-row {
+  display: flex;
+  align-items: center;
+}
+.realtime-string {
+  .content-wrapper {
+    display: flex;
+    flex-direction: column;
+  }
+  textarea {
+    resize: none;
+    border: none;
+  }
+  h4 {
+    margin-top: 0;
+  }
+  .title {
+    background: none;
+  }
+  .editing-message {
+    font-style:italic;
+    margin-top:-30px;
+    margin-left:30%;
+    color:orange
+  }
+  .treatment-name {
+    height:auto;
+  }
+}
+.summary-bottom {
```

Comment:
Not necessary to create a class with one property unless the class is enabled with a condition. The same applies to margin-bottom-0

Reply:
I don't want to write css in html and also this class will be used at multiple places